src/sql-parser/src/parser.rs

```diff
@@ -1859,6 +1861,29 @@ impl<'a> Parser<'a> {
         Ok(compression)
     }
 
+    fn parse_create_connector(&mut self) -> Result<Statement<Raw>, ParserError> {
+        self.expect_keyword(CONNECTOR)?;
+        let if_not_exists = self.parse_if_not_exists()?;
+        let name = self.parse_object_name()?;
+        self.expect_keyword(FOR)?;
+        let connector = if self.parse_keyword(KAFKA) {
```

Comment:
I think this can use `expect_keyword`, which obviates the need for the else branch

Reply:
I was trying to anticipate when I add the ConfluentSchemaRegistry connector variant where this will either become a match or get additional if else conditions.

Reply:
Oh sure--didn't realize that work was imminent. nbd.